torch/nn/quantization_test.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for quantization.py."""
+# Run using `python -m torch.nn.quantization_test`
+
+import unittest
+
+from .quantization import quantize, get_max_signed_int, get_max_unsigned_int
+from .test_utils import arange_nd, TorchTestCase
+
+import torch
+
+
+class QuantizationTest(TorchTestCase):
+
+  def test_bits_none(self):
```

Comment:
It will be nice to add test for individual functions as well, for example scaled_inputs; round and clip and tests for STE.